src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -116,7 +116,7 @@ protected function isPropertyEnabled(string $property): bool
      */
     protected function isPropertyMapped(string $property, string $resourceClass, bool $allowAssociation = false): bool
     {
-        if ($this->isPropertyNested($property)) {
+        if ($this->isPropertyNested($property) && !$this->isPropertyNestedEmbedded($property, $resourceClass)) {
```

Comment:
Yes, we also need to fix `extractProperties`. `$needsFixing = true` for embedded fields. :smile: